sync2jira/main.py

```diff
@@ -214,12 +232,25 @@ def initialize_issues(config, testing=False, repo_name=None):
     log.info("Running initialization to sync all issues from upstream to jira")
     log.info("Testing flag is %r", config["sync2jira"]["testing"])
     mapping = config["sync2jira"]["map"]
+    for upstream in mapping.get("pagure", {}).keys():
+        if "issue" not in mapping.get("pagure", {}).get(upstream, {}).get("sync", []):
+            continue
+        if repo_name is not None and upstream != repo_name:
+            continue
+        for issue in u_issue.pagure_issues(upstream, config):
+            try:
+                d_issue.sync_with_jira(issue, config)
+            except Exception as e:
+                log.error(f"Failed on {issue}\nException: {e}")
+                raise
```

Comment:
As a general practice, you should use `%s`-style format specifiers (instead of f-strings) for log messages, passing the values as arguments, because this defers the formatting operation until after the level filtering:
```suggestion
            except Exception as e:
                log.error("Failed on %r\nException: %s", issue, str(e))
                raise
```